sysutils/upt/Portfile

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                upt
+version             0.8
+revision            0
+
+platforms           darwin
+supported_archs     noarch
+categories          sysutils python
+license             BSD
+maintainers         {@korusuke somaiya.edu:karan.sheth} openmaintainer
+
+description         Package software from any package manager to any distribution
+long_description    ${description}
+
+homepage            https://framagit.org/upt/upt
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            ${python.rootname}-${version}
+
+checksums           sha256  ecb4b8da0e2e6a5f6926dc41cc45fd3923e8dfb6a9a8aafaa9149b5144d8898c \
+                    rmd160  3d3aade4ec513943b4fc8f12042a3f4328f60102 \
+                    size    25792
+
+python.default_version  37
+
+if {${name} ne ${subport}} {
```

Comment:
since we are not using it as a module anymore and, therefore, don't have the ```python.versions``` anymore, you will need to remove this ```if```-block. You probably noticed also locally (and as reported by the CI tests) that right now ```upt``` does not build.

Reply:
I am currently not able to build ports locally, therefore all these errors.